plone/app/discussion/interfaces.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
 """Interfaces for plone.app.discussion
 """
+from plone import api
```

Comment:
plone.api is - unfortunately - not allowed in Plone core packages. This is in order to avoid circular dependencies.